config-base-next.js

```diff
@@ -20,5 +21,17 @@ module.exports = {
         'unicorn/no-null': 'warn',
       },
     },
+    {
+      files: ['**/test/**/*'],
+      rules: {
+        '@typescript-eslint/consistent-type-assertions': 'off',
+        '@typescript-eslint/explicit-function-return-type': 'off',
+        '@typescript-eslint/explicit-module-boundary-types': 'off',
+        '@typescript-eslint/no-empty-function': 'off',
+        '@typescript-eslint/require-await': 'off',
```

Comment:
I don't think we should be turning any of these rules off here. Do you know why they're being disabled?

Reply:
not sure, these rules and majority of the other ones were already set to `off` in @MaxGoo 's PR that this is based off of

Reply:
I mean I tested these all at one point and for one reason or another determined we don't want it. We have common patterns that do these things and I didn't want to introduce the need to ignore these rules in-line all over the place. But I don't specifically remember any of these.

If we want them on (or either way) part of nailing this down (and taking it over from me) should be testing to make sure the configs are reasonable to implement in a few services.

So idonno if you've done this yet @AD-Blue , but ideally you should link this package locally and try makinf the change in a few services to make sure they are reasonable. So if we want to turn these to on that's what you'll have to do.

Reply:
is there a way to quickly test changes locally without publishing a `next` canary version? That's how I've been testing but I've had to publish each time

Reply:
That's probably the best way. You can also just use `npm pack` to create a zipfile and then install that in another project.

Reply:
https://docs.npmjs.com/cli/v9/commands/npm-link

or if that's not working you can just run build in `eslint-config-neo` and then use that build folder to replace the build that's in node_modules in the service you are testing in.

or you can just link to a local file (after you build it) in your package.json. Not sure of the exact syntax, but something like this.
```
...
eslint-config-neo:file:../eslint-config-neo/build
...
```

Reply:
yea I think I was doing the last option, where you run npm pack in eslint-config-neo and then just use that in the package.json of the service you are testing in.

updated my above comment because i forgot I used `npm pack` rather than just doing a build